Resources/Prototypes/Entities/Objects/Specific/Librarian/books_bag.yml

```
@@ -23,4 +23,8 @@
         tags:
           - Book
           - Document
+          - Write
+          - Dice
+          - Figurine
+          - TabletopBoard
```

```suggestion
          - Book
          - Dice
          - Document
          - Figurine
          - TabletopBoard
          - Write
```

alphabetical makes it way easier to work with.

---

Sounds good to me